src/metakb/load_data.py

```diff
@@ -479,14 +479,22 @@ def _add_statement_evidence(tx: ManagedTransaction, statement_in: dict) -> None:
     if strength:
         strength_key_fields = ("primaryCode", "label")
 
-        strength_keys = _create_parameterized_query(
-            strength, strength_key_fields, entity_param_prefix="strength_"
-        )
+        strength_keys = [
+            _create_parameterized_query(
+                strength, strength_key_fields, entity_param_prefix="strength_"
+            )
+        ]
         for k in strength_key_fields:
             v = strength.get(k)
             if v:
                 statement[f"strength_{k}"] = v
 
+        mappings = strength.get("mappings", [])
+        if mappings:
+            statement["strength_mappings"] = json.dumps(mappings)
+            strength_keys.append("mappings:$strength_mappings")
```

Comment:
this line is confusing me. Is this just acting as a placeholder?

Reply:
This allows us to create both the strength node and statement node. Line 494 allows the transaction to access the strength values. The following lines includes the key/value pairs needed for line 499